src/context/mod.rs

```diff
@@ -3,25 +3,43 @@
 //! An _execution context_ represents the current state of a thread of execution in the
 //! JavaScript engine. Internally, it tracks things like the set of pending function calls,
 //! whether the engine is currently throwing an exception or not, and whether the engine is
-//! in the process of shutting down. Abstractly, however, the context also manages what
-//! operations are safely available.
+//! in the process of shutting down. The context uses this internal state to manage what
+//! operations are safely available and when.
 //!
 //! The [`Context`](Context) trait provides an abstract interface to the JavaScript
 //! execution context. All interaction with the JavaScript engine in Neon code is mediated
 //! through instances of this trait.
 //!
+//! One particularly useful context type is [`CallContext`](CallContext), which is passed
+//! to all Neon functions as their initial execution context (or [`FunctionContext`](FunctionContext),
+//! a convenient shorthand for `CallContext<JsObject>`):
+//!
+//! ```ignore
```

Comment:
Nit, this one will compile on both backends and doesn't need to be `ignore`.

```suggestion
//! ```
# use neon::prelude::*;
```